### PR TITLE
upgrade kitematic Casks

### PR DESCRIPTION
**Pull request by FinalDes**

### CODE REVIEW

File: Casks/kitematic.rb [hunk 1]
```diff
@@ -9,4 +9,9 @@
   license :affero
 
   app 'Kitematic (Beta).app'
+
+  zap :delete => [
+	'~/Library/Application\ Support/Kitematic',
```
Comment (vitorgalvao) [inline Casks/kitematic.rb]: `\` shouldn’t be there. The path is quoted.

Comment (FinalDes) [inline Casks/kitematic.rb]: can explain more detail?

Comment (vitorgalvao) [inline Casks/kitematic.rb]: `'~/Library/Application\ Support/Kitematic'` should be `'~/Library/Application Support/Kitematic'` (remove the `\`).

**Comment by vitorgalvao** [comment]
You’re using tabs for indentation. Please change them to spaces.

**Comment by vitorgalvao** [comment]
Superseded by #10020.

**Pull request closed without merging**